src/components/toolbar/__tests__/__snapshots__/toolbar.test.js.snap

```diff
@@ -157,7 +157,7 @@ Array [
       Object {
         "data": Object {
           "clearFilters": Object {
-            "sla": null,
+            "usage": null,
```

Comment:
required:
double check why this snapshot shifted, may have to do with the "default" on a switch

Reply:
It's related to `"type": "SET_QUERY_CLEAR"` and appears to be valid, it's unclear why it updated now, we'll note and keep an eye on it

Reply:
Figured it out... the issue was test specific not on the component or helpers, around line 83 https://github.com/RedHatInsights/curiosity-frontend/blob/ci/src/components/toolbar/__tests__/toolbar.test.js#L83

We were doing a "find" against a borked string... then we made more helpful snapshots

before
```
console.log src/components/toolbar/__tests__/toolbar.test.js:85
      CHECK >>> onSelect usage t(curiosity-toolbar.category, [object Object])
```
after
```
console.log src/components/toolbar/__tests__/toolbar.test.js:85
      CHECK >>> onSelect usage t(curiosity-toolbar.category, {"context":"usage"})
```
And everything started working... apparently the test was broken before =(